startup_scripts/470_contact_roles.py

```diff
@@ -0,0 +1,27 @@
+import sys
+
+from startup_script_utils import (
+    load_yaml,
+    pop_custom_fields,
+    set_custom_fields_values,
+    split_params,
+)
+from tenancy.models import ContactRole
+
+contact_roles = load_yaml("/opt/netbox/initializers/contact_roles.yml")
+
+if contact_roles is None:
+    sys.exit()
+
+match_params = ["name", "slug"]
+
+for params in contact_roles:
+    custom_field_data = pop_custom_fields(params)
+
+    matching_params, defaults = split_params(params, match_params)
```

Comment:
Same comment as above.
```suggestion
    matching_params, defaults = split_params(params)
```